ConwaysGameOfLife/Generator.cs

```diff
@@ -29,19 +30,32 @@ rules checked
             outputformatter
         */
         Universe _universe;
+        IOutput _output;
 
-        
 
         public Generator(Universe universe)
         {
             _universe = universe;
-            
+            _output = new ConsoleOutput();

```

Comment:
This will probably need to be passed in as a parameter to `Generator` so you can inject your `StubOutput` when testing

Reply:
This is called dependency injection - some [fun reading ](https://alexalvess.medium.com/dependency-injection-and-inversion-of-control-on-net-core-3136fe98b72) to give you a bit more info